docs/src/icons/SvgMuiLogo.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+function SvgComponent(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
```

Comment:
Just a heads-up for us to use this logo as the new favicon too :) Couldn't add a comment on the file where we declare it but yeah!

Reply:
updated.